### PR TITLE
Personalises the application interface to the user that is currently logged in

### PR DESCRIPTION
**Pull request by nickwxo**
This branch personalises the application interface to the user that is currently logged in and properly implements Cam's designs for the same.

### CODE REVIEW

File: routes/user/router-user.js [hunk 1]
```diff
@@ -14,6 +14,8 @@ var express = require('express'),
 router.get('/user/:id', User.getUser);
 router.get('/users', User.getAllUsers);
 router.post('/user', User.createUser);
+router.get('/currentuser', User.getCurrentUser);
```
Comment (matjames007) [inline routes/user/router-user.js]: @nick22891 change this endpoint to `GET /user`

File: public/partials/common/navigation-loggedin.html [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<!-- Brand and toggle get grouped for better mobile display -->
```
Comment (matjames007) [inline public/partials/common/navigation-loggedin.html]: @nick22891 I am not a big fan of splitting the navigation file into 3 parts...

**Comment by nickwxo** [comment]
@matjames007 You can also review this one again when you get a chance - I've made the changes.

**Comment by matjames007** [comment]
@nick22891 review...

**Comment by nickwxo** [comment]
Reviewed, tested and works fine.

**Pull request merged**